zh-cn/go-cn.html.markdown

```diff
@@ -36,7 +40,7 @@ import (
 func main() {
     // 往标准输出打印一行。
     // 用包名fmt限制打印函数。
-    fmt.Println("Hello world!")
+    fmt.Println("天坑欢迎你!")
```

Comment:
I think this is not suitable, these words have negative meanings.

Reply:
please send a new PR then